.github/workflows/codeql.yml

```
@@ -12,9 +12,9 @@
 name: "CodeQL Advanced"
 on:
   push:
-    branches: ["main", "stage", "feature/*", "hotfix/*", "bugfix/*", "release/*", "stable/*"]
+    branches: ["release/*"]
```

```suggestion
    branches: ["release/*", "main"]
```
We should be targetting "main" as well just so we are ensuring code scanning and other checks upon deployment to production